scripts/exgdas_atmos_chgres_forenkf.sh

```diff
@@ -18,11 +18,9 @@
 #
 ################################################################################
 
-#  Set environment.
-export VERBOSE=${VERBOSE:-"YES"}
-if [ $VERBOSE = "YES" ]; then
-   echo $(date) EXECUTING $0 $* >&2
-   set -x
+PREAMBLE_SCRIPT="${PREAMBLE_SCRIPT:-$HOMEgfs/ush/preamble.sh}"
+if [ -f "${PREAMBLE_SCRIPT}" ]; then
+  source $PREAMBLE_SCRIPT
```

Comment:
Why is this needed in the exscript when the JJob has already set the necessary tracing, error handling etc.
This seems like a lot of duplication.

Reply:
The settings aren't handed down to daughter scripts (unless sourced, of course).

Reply:
Also, even if the `set` built-in were, the way I have the postamble set up with the trap wouldn't work.

Reply:
to be evaluated later.